release_notes/ocp-4-2-release-notes.adoc

```diff
@@ -0,0 +1,890 @@
+
+|Prometheus Cluster Monitoring
+|GA
+|GA
+|GA
+
+|Local Storage Persistent Volumes
+|TP
+|TP
+|TP
+
+|CRI-O for runtime pods
+|GA* footnoteref:[disclaimer, Features marked with `*` indicate delivery in a z-stream patch.]
+|GA
+|GA
+
+|Tenant Driven Snapshotting
+|TP
+|TP
+|TP
+
+|`oc` CLI Plug-ins
+|TP
+|TP
+|TP
+
+|Service Catalog
+|GA
+|GA
+|--
+
+|Template Service Broker
+|GA
+|GA
+|--
+
+|OpenShift Ansible Service Broker
+|GA
+|GA
+|--
+
+|Network Policy
+|GA
+|GA
+|GA
+
+|Multus
+|-
+|GA
+|GA
+
+|Service Catalog Initial Experience
+|GA
+|GA
+|GA
+
+|New Add Project Flow
+|GA
+|GA
+|GA
+
+|Search Catalog
+|GA
+|GA
+|GA
+
+|Cron Jobs
+|GA
+|GA
+|GA
+
+|Kubernetes Deployments
+|GA
+|GA
+|GA
+
+|StatefulSets
+|GA
+|GA
+|GA
+
+|Explicit Quota
+|GA
+|GA
+|GA
+
+|Mount Options
+|GA
+|GA
+|GA
+
+|System Containers for Docker, CRI-O
+|-
+|-
+|-
+
+|Hawkular Agent
+|-
+|-
+|-
+
+|Pod PreSets
+|-
+|-
+|-
+
+|experimental-qos-reserved
+|TP
+|TP
+|TP
+
+|Pod sysctls
+|GA
+|GA
+|GA. See xref:ocp-4-2-known-issues[Known issues] for current limitations.
+
+|Central Audit
+|GA
+|GA
+|GA
+
+|Static IPs for External Project Traffic
+|GA
+|GA
+|GA
+
+|Template Completion Detection
+|GA
+|GA
+|GA
+
+|`replicaSet`
+|GA
+|GA
+|GA
+
+|Fluentd Mux
+|TP
+|TP
+|TP
+
+|Clustered MongoDB Template
+|-
+|-
+|-
+
+|Clustered MySQL Template
+|-
+|-
+|-
+
+|ImageStreams with Kubernetes Resources
+|GA
+|GA
+|GA
+
+|Device Manager
+|GA
+|GA
+|GA
+
+|Persistent Volume Resize
+|GA
+|GA
+|GA
+
+|Huge Pages
+|GA
+|GA
+|GA
+
+|CPU Pinning
+|GA
+|GA
+|GA
+
+|Admission Webhooks
+|TP
+|GA
+|GA
+
+|External provisoner for AWS EFS
+|TP
+|TP
+|TP
+
+|Pod Unidler
+|TP
+|TP
+|TP
+
+|Node Problem Detector
+|TP
+|TP
+|TP
+
+|Ephemeral Storage Limit/Requests
+|TP
+|TP
+|TP
+
+|Descheduler
+|TP
+|TP
+|TP
+
+|CephFS
+|TP
+|TP
+|TP
+
+|Podman
+|TP
+|TP
+|TP
+
+|Kuryr CNI Plug-in
+|TP
+|TP
+|TP
+
+|Sharing Control of the PID Namespace
+|TP
+|TP
+|TP
+
+|Manila Provisioner
+|TP
+|TP
+|TP
+
+|Cluster Administrator console
+|GA
+|GA
+|GA
+
+|Cluster Autoscaling (AWS Only)
+|GA
+|GA
+|GA
+
+|Container Storage Interface (CSI)
+|TP
+|TP
+|GA
+
+|Operator Lifecycle Manager
+|TP
+|GA
+|GA
+
+|Red Hat OpenShift Service Mesh
+|TP
+|TP
+|TP
+
+|"Fully Automatic" Egress IPs
+|GA
+|GA
+|GA
+
+|Pod Priority and Preemption
+|GA
+|GA
+|GA
+
+|Multi-stage builds in Dockerfiles
+|TP
+|GA
+|GA
+
+|OVN
+|
+|TP
+|TP
+
+|OVN SDN
+|
+|TP
+|TP
+
+|HPA custom metrics adapter based on Prometheus
+|
+|TP
+|TP
+
+|Machine health checks
+|
+|TP
+|TP
+
+|Raw Block with iSCSI
+|
+|
+|TP
+
+|====
+
+[id="ocp-4-2-known-issues"]
+== Known issues
+
+* Unsafe sysctl cannot be used in {product-title} 4.2.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1690754[*BZ#1690754*])
+
+* `git clone` operations that go through a proxy that is performing man-in-the-middle
+(MITM) TLS hijacking or reencrypting of the proxied connection will not work.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1750650[*BZ#1750650*])
```

Comment:
after much discussion with @ricardomaraschini and @gabemontero i believe the restriction is broader than this, or perhaps better stated as:  "`git clone` operations that go through an HTTPS proxy will fail.  Non-TLS (HTTP) proxies can be used successfully" (link to bug is correct)